main.go

```diff
@@ -124,15 +124,19 @@ func setupLogging(isDebug bool, logLevel, logFormat string) error {
 }
 
 // createPrometheusRegistry creates and configures a Prometheus registry
-func createPrometheusRegistry(redisMetricsOnly bool) *prometheus.Registry {
+func createPrometheusRegistry(redisMetricsOnly, extendedGoMetrics bool) *prometheus.Registry {
```

Comment:
gotta rename this if we go with "runtime metrics"

Reply:
Yep, done